eb

```diff
@@ -80,12 +80,12 @@ for python_cmd in ${EB_PYTHON} ${EB_INSTALLPYTHON} 'python' 'python3' 'python2';
 
         if [ ! -z $PYTHON ]; then
             # check whether easybuild.main is available for selected python command
```

Comment:
Probably should replace `easybuild.main` in this comment by `easybuild` in order not to create confusion